src/marshaller/HipieDDL.js

```diff
@@ -291,6 +299,9 @@
             case "CHORO":
                 this.mappings = new ChoroMappings(this.visualization, source.mappings, source.link);
                 break;
+            case "HEAT_MAP":
```

Comment:
Again this will be renamed to be more generic - IOW its the data shape name, rather than the viz name at this point...